api/HTMLCanvasElement.json

```diff
@@ -185,171 +185,62 @@
                 "version_added": "6.0"
               },
               "webview_android": {
-                "version_added": false
+                "version_added": "56"
               }
             },
             "status": {
               "experimental": false,
               "standard_track": true,
               "deprecated": false
             }
-          }
-        },
-        "desynchronized": {
-          "__compat": {
-            "description": "<code>desynchronized</code> attribute",
-            "spec_url": "https://www.khronos.org/registry/webgl/specs/latest/1.0/#WEBGLCONTEXTATTRIBUTES",
-            "support": {
-              "chrome": [
-                {
-                  "version_added": "81",
-                  "notes": "ChromeOS and Windows"
+          },
+          "options_alpha_parameter": {
+            "__compat": {
+              "description": "<code>options.alpha</code> parameter",
+              "spec_url": "https://www.khronos.org/registry/webgl/specs/latest/1.0/#WEBGLCONTEXTATTRIBUTES",
+              "support": {
+                "chrome": {
+                  "version_added": "56"
                 },
-                {
-                  "version_added": "75",
-                  "notes": "ChromeOS only"
+                "chrome_android": {
+                  "version_added": false
```

Comment:
This had come from the original data for this option, but I had a sneaking suspicion it might have been incorrect.  I haven't verified this data yet, however.

Reply:
Ran a test using the following code and found there is indeed Android support.  Updating!

```html
<div id="test">
	<canvas id="canvas"></canvas>
</div>

<script>
	var canvas = document.getElementById('canvas');
	var ctx = canvas.getContext('2d', {alpha: true});

	ctx.fillStyle = "#ff0000aa";
	ctx.fillRect(10, 10, 100, 100);
</script>
```